roles/build-release/defaults/main.yaml

```diff
@@ -59,6 +59,17 @@ antsibull_tags_enforce_policy: false
 # File containing a newline separated list of collections to ignore
 antsibull_tags_ignores_file: "{{ antsibull_data_dir }}/validate-tags-ignores"
 
+
+#####
+# These variables relate to testing bytecompilation of the Python code in the
```

Comment:
It should be byte-compile, not bytecompile. Same in many other places :)
```suggestion
# These variables relate to testing byte-compilation of the Python code in the
```

Reply:
I hyphenated `bytecompile` in comments and help text but left the variable names the same (I prefer `bytecompile` over `byte_compile`). Wdyt?

Reply:
I agree, for variable names `bytecompile` is fine!